source/features/rgh-options-link.tsx

```diff
@@ -0,0 +1,34 @@
+/* You can have invisible/inactive links to the options page on the wiki. This feature enables them. */
```

Comment:
Two changes:

- it now applies to the entire wiki, and maybe the entire repo in the future
- it simplifies the markup while remaining invisible without extension
  
  ```diff
  - [](#rgh-linkify-welcome-issue)extension’s options[](#rgh-linkify-welcome-issue)
  + <a name="options-page-link">extension’s options</a>
  ```